examples/server/server.cpp

```diff
@@ -599,10 +624,31 @@ int main(int argc, const char ** argv) {
         res_ok_json(res, health);
     };
 
+    const auto handle_voices = [&args, &tqueue, &rmap, &res_error, &res_ok_voices](const httplib::Request & req, httplib::Response & res) {
+        struct simple_server_task * task = new simple_server_task(VOICES);
+        int id = task->id;
+        tqueue->push(task);
+        struct simple_server_task * rtask = rmap->get(id);
+        if (!rtask->success) {
+            json formatted_error;
+            if (has_prefix(rtask->message, "Voices are not supported")) {
+                formatted_error = format_error_response(rtask->message, ERROR_TYPE_NOT_SUPPORTED);
+            } else {
+                formatted_error = format_error_response(rtask->message, ERROR_TYPE_SERVER);
+            }
+            res_error(res, formatted_error);
+            return;
+        }
+        std::vector<std::string> voices = split(rtask->message, ",");
```

Comment:
This is very inefficient and a more generic response pattern rather than parsing a string list from the message.